dmscripts/helpers/supplier_data_helpers.py

```diff
@@ -17,33 +20,43 @@ class SupplierFrameworkData(object):
 
     data = None
 
-    def __init__(self, client, target_framework_slug):
+    def __init__(self, client, target_framework_slug, *, supplier_ids=None):
+        """
+        :param client: Client object for the Digital Marketplace API
+        :param target_framework_slug str: Framework slug
+        :param supplier_ids: List of supplier IDs to filter data by
+        """
         self.client = client
         self.target_framework_slug = target_framework_slug
+        self.supplier_ids = supplier_ids
 
-    def get_supplier_frameworks(self):
+    def get_supplier_frameworks(self, supplier_ids=None):
         """Return supplier frameworks."""
-        return self.client.find_framework_suppliers(
+        framework_suppliers = self.client.find_framework_suppliers(
```

Comment:
It's prrrrrobably worth using `find_framework_suppliers_iter` here in case we ever page-ify this endpoint.

Reply:
Done